### PR TITLE
Add Brazilian power outlet standard to choices.py

### LINKED ISSUE #13009: Add Brazilian power outlet standard
### NetBox version

v3.5.4

### Feature type

Data model extension

### Proposed functionality

Add Brazilian power outlet standard as a power outlet type.

### Use case

Power outlets in Brazil are standardized by the Brazilian Association of Technical Standards (ABNT - Associação Brasileira de Normas Técnicas).
Our outlets were standardized in 2006 and have been enforced since 2011, making every manufacture that wishes to sell its products within national territory to comply with it. The standard is called [NBR 14136](https://en.wikipedia.org/wiki/IEC_60906-1#Brazilian_NBR_14136_standard_) and is based on the [IEC 60906-1](https://pt.wikipedia.org/wiki/IEC_60906-1) standard.

### Database changes

_No response_

### External dependencies

_No response_

### PR DESCRIPTION
**Pull request by BrunoBlanes**
### Fixes: #13009

Add 10A and 20A plugs standardized by the Brazilian Association of Technical Standards.

### CODE REVIEW

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: netbox/dcim/choices.py [hunk 1]
```diff
@@ -724,6 +727,10 @@ class PowerOutletTypeChoices(ChoiceSet):
             (TYPE_ITA_O, 'ITA Type O'),
             (TYPE_ITA_MULTISTANDARD, 'ITA Multistandard'),
         )),
+        ('NBR 14136', (
```
[the remark this answers is not in the document]
Comment (BrunoBlanes) [reply netbox/dcim/choices.py:730]: This makes sense, we can group all plug types derived from this IEC to make it clearer.

**Comment by jeremystretch** [comment]
This defines the outlet (female) types but not the port (male) types. @BrunoBlanes could you add those too please?

**Comment by BrunoBlanes** [comment]
> This defines the outlet (female) types but not the port (male) types. @BrunoBlanes could you add those too please?

Sure.

**Comment by jeremystretch** [comment]
To be honest I'm not 100% on the "2P+T" descriptor as I don't see it documented anywhere, but I'll defer to @BrunoBlanes as a regional user who is no doubt more familiar with the standard.

**Comment by BrunoBlanes** [comment]
> To be honest I'm not 100% on the "2P+T" descriptor as I don't see it documented anywhere, but I'll defer to @BrunoBlanes as a regional user who is no doubt more familiar with the standard.

It does not show up as an acronym on the standard, but vendors did make a good effort of putting it everywhere on their products to the point that it has become a standard itself.

**Comment by jeremystretch** [comment]
Thanks @BrunoBlanes!

**Pull request merged**